app/controllers/foods_controller.rb

```diff
@@ -13,12 +13,11 @@ def create
     return unless @food.save!
 
     flash.now[:notice] = 'Food created successfully'
-    redirect_to root_path
+    redirect_to foods_path
   end
 
   def destroy
     @food = Food.find(params[:id])
-    # @user = @post.author_id
```

Comment:
As mentioned above kindly remove this commented code is not considered a good practice and makes your project look untidy and not organized.